app/views/transaction-detail/_refund.njk

```diff
@@ -1,5 +1,4 @@
-
-<form id="refundForm" action="/transactions/{{charge_id}}/refund" method="post" class="target-to-show {% if flash.genericError %}active{% endif %}">
+<form id="refundForm" action="{{refundUrl}}" method="post" class="target-to-show {% if flash.genericError %}active{% endif %}">
```

Comment:
`formatAccountPathsFor` is available to the template, potentially using it here would reduce the number of things needed to be passed in to the template 

```suggestion
<form id="refundForm" action="{{ formatAccountPathsFor(routes.account.transactions.refund, currentGatewayAccount.external_id, charge_id) }}" method="post" class="target-to-show {% if flash.genericError %}active{% endif %}">
```

Reply:
Good spot - done.